src/display.cpp

```diff
@@ -233,6 +233,8 @@ display::display(const display_context * dc, std::weak_ptr<wb::manager> wb, repo
 
 	set_idle_anim_rate(preferences::idle_anim_rate());
 
+	if(preferences::tile_size() > 0)
+		zoom_ = preferences::tile_size();
```

Comment:
The setter function does some validation, under `Confirm this is indeed a valid zoom level`. Is it needed here, too? I realize that such values can't be written into the preferences file by the setter, but someone might edit their preferences file manually, or we might change the validation between versions, etc.

Reply:
The potential for preferences file editing/corruption is why I added the check, in non-positive cases the zoom should fall back to the default 72 (which appeared to be the case when I tested).